Package doesn't build on Raspbian 8

I installed the deps specified in REQUIREMENTS; dpkg-builddeps returns nothing (ok). Fresh raspbian 8 install

```
pi@raspberrypi:~/gst/gst-rpicamsrc $ sudo apt-get install autoconf automake libtool pkg-config libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libraspberrypi-dev
pi@raspberrypi:~/gst/gst-rpicamsrc $ dpkg-buildpackage 
dpkg-buildpackage: source package gstreamer1.0-rpicamsrc
dpkg-buildpackage: source version 1.0.0
dpkg-buildpackage: source distribution wheezy
dpkg-buildpackage: source changed by hornetwl <hornetwlfpv@googlemail.com>
dpkg-buildpackage: host architecture armhf
 dpkg-source --before-build gst-rpicamsrc
 fakeroot debian/rules clean
dh clean  --with autotools-dev
   dh_testdir
   dh_auto_clean
   dh_autotools-dev_restoreconfig
   dh_clean
 dpkg-source -b gst-rpicamsrc
dpkg-source: info: using source format `3.0 (native)'
dpkg-source: info: building gstreamer1.0-rpicamsrc in gstreamer1.0-rpicamsrc_1.0.0.tar.xz
dpkg-source: info: building gstreamer1.0-rpicamsrc in gstreamer1.0-rpicamsrc_1.0.0.dsc
 debian/rules build
dh build  --with autotools-dev
   dh_testdir
   dh_autotools-dev_updateconfig
   dh_auto_configure
   dh_auto_build
   dh_auto_test
 fakeroot debian/rules binary
dh binary  --with autotools-dev
   dh_testroot
   dh_prep
   debian/rules override_dh_auto_install
make[1]: Entering directory '/home/pi/gst/gst-rpicamsrc'
dh_auto_install
find /home/pi/gst/gst-rpicamsrc/debian/gstreamer1.0-rpicamsrc -name '*.la' | xargs rm --verbose
rm: missing operand
Try 'rm --help' for more information.
debian/rules:28: recipe for target 'override_dh_auto_install' failed
make[1]: *** [override_dh_auto_install] Error 123
make[1]: Leaving directory '/home/pi/gst/gst-rpicamsrc'
debian/rules:25: recipe for target 'binary' failed
make: *** [binary] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary gave error exit status 2
```

--- a/debian/rules
+++ b/debian/rules
@@ -26,5 +26,5 @@ TMP     = $(CURDIR)/debian/$(PACKAGE)
 
 override_dh_auto_install:
 	dh_auto_install
-	find $(TMP) -name '*.la' | xargs rm --verbose
+	find $(TMP) -name '*.la' | xargs rm --verbose || true
 

